src/components/Video/Video.jsx

```diff
@@ -28,29 +64,31 @@ function Video({ video }) {
           <h2 className="video-title">{video.title}</h2>
           <div className="video-info fs-sm">{video.views} | {video.publishDate}</div>
           <div className="video-btns">
-            <button className="btn btn-active btn-video">
+
+            <button onClick={likef}
+              className={[likeactive ? "active-like" : null, "count_btn"].join(" ")}>
```

Comment:
instead of taking an array and using `.join()`, you can just write `className={likeactive ? "active-like count_btn" : "count_btn"}`